packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx

```diff
@@ -118,6 +119,7 @@ export class AtomicQuerySummary implements InitializableComponent {
     }
 
     return (
+      //TODO: remove renderNoResults method
```

Comment:
If these TODOs are due to deprecation, I would encourage annotating with `deprecated` instead. This will help search for them, and distinguish them from other TODOs,

```
/**
* @deprecated
*/
```